options.cpp

```diff
@@ -49,6 +49,24 @@ RtSmithOptions::RtSmithOptions()
             return true;
         },
         "Write the generated config to the specified .txtpb file.");
+    registerOption(
+        "--user-p4info", "filePath",
+        [this](const char *arg) {
+            _userP4Info = arg;
+            if (!std::filesystem::exists(_userP4Info.value())) {
+                ::error("%1% does not exist. Please provide a valid file path.",
+                        _userP4Info.value().c_str());
+                return false;
+            }
+            if (_p4InfoFilePath.has_value()) {
+                ::error(
+                    "Both --user-p4info and --generate-p4info are specified. Please specify only "
+                    "one.");
+                return false;
+            }
+            return true;
+        },
+        "Write the P4Runtime control plane API description (P4Info) to the specified .txtpb file.");
```

Comment:
This is probably copied but this description is wrong.

Reply:
Ah yeah. So the one in Flay is also wrong. I will also push a fix for the one in Flay later.

Reply:
Yes